pkg/test/env/istio.go

```diff
@@ -89,6 +89,9 @@ var (
 
 	CrdsFilesDir = path.Join(ChartsDir, "istio-init/files")
 
+	// InstallDir specifies the base directory for the alpha installer
+	InstallDir = path.Join(GOPATH.ValueOrDefault(build.Default.GOPATH), "/src/istio.io/installer")
```

Comment:
Doesn't build.Default.GOPATH already take the environment variable into the account?

Also the installer repo is not a Go repo, there is no reason it should be checked out under GOPATH, and with modules GOPATH has even less meaning. Shouldn't this be controlled by some env var/flag?